README.md

```diff
@@ -31,29 +54,32 @@ may include:
 
 - Adding a new service that enables new ways of managing datasets
 
-## How to submit an RFC
+## How to submit a feature pitch or an RFC
 
-- Copy the `rfc-template.md` file and name it like `0000-my-feature.md` (with an
-  incrementing number and appropriate title). Make a subdirectory in the
-  `design/` directory for your RFC.
+- Copy the appropriate template (`feature-pitch-template.md` or
+  `rfc-template.md`) and name it like `0000-my-feature.md` (with an incrementing
```

Comment:
The added pitch does not follow this numbered template.

Reply:
True - I've never known whether this numbering scheme actually makes sense or not...

Reply:
It doesn't matter to me whether we do it or not, just that we're consistent with whatever we write in our own documentation :)

If I had to pick, the numbering helps with more quickly finding the latest one (which is usually what one is looking for).

Reply:
done